documentation/wiki/ChangeWaves.md

```diff
@@ -24,6 +24,9 @@ A wave of features is set to "rotate out" (i.e. become standard functionality) t
 
 ## Current Rotation of Change Waves
 
+### 18.0
```

Comment:
I used 18.0 because I saw that #12350 will add a change wave on that version too.